modules/social_features/social_group/modules/social_group_invite/social_group_invite.module

```diff
@@ -545,3 +546,33 @@ function social_group_invite_mail_alter(&$message) {
     $language_manager->setConfigOverrideLanguage($original_language);
   }
 }
+
+/**
+ * Implements hook_tokens_alter().
+ */
+function social_group_invite_tokens_alter(array &$replacements, array $context, BubbleableMetadata $bubbleable_metadata) {
+  if (
+    !empty($replacements) &&
+    isset($context["data"]["group"]) &&
+    $context["type"] === 'group_content'
+  ) {
```

Comment:
Add comments about what and why we do here.

Reply:
done